File: resources/Torus World/scripts/Moving.js
stdout.println( "Loading Moving.js");
Moving = Class({
    Extends: RoleScript,
    
    init: function() {
    	this.vx = 0;
    	this.vy = 0;
    },

    tick: function() {
        this.actor.moveBy(this.vx, this.vy);
    }
});

File: resources/Pyrus World/scripts/moving.py
from uk.co.nickthecoder.itchy import AbstractRole

class Moving(AbstractRole) :

    def __init__(self) :
        self.vx = 0
        self.vy = 0

    def tick(self) :
        actor = self.getActor()
        actor.moveBy(self.vx, self.vy)

